lib/src/main/java/org/team100/lib/motor/drive/Falcon6DriveMotor.java

```diff
@@ -231,12 +218,13 @@ public void setVelocity(double outputM_S, double accelM_S_S) {
 
         VelocityDutyCycle v = new VelocityDutyCycle(motorRev_S);
```

Comment:
the drive motor uses VelocityDutyCycle but the turning motor uses VelocityVoltage,and really the best option is VelocityTorqueCurrentFOC.